### PR TITLE
Add Support for Genero Business Development Language

### PR DESCRIPTION
**Pull request by alienriver49**
Add support for Genero and Genero Forms. The grammars are different for these two "languages" which is why they have been separated out. Genero Forms is a markup for Genero which allows the rendering of Genero programs in Windows, Android, iOS, or the Web. There was no need to add heuristics as there were no languages using either the 4gl or per extension.

## Description
Adding support for the Genero Business Development Language from [FourJs](http://www.4js.com/). This language is the evolution of the language Informix 4GL. There is still a lot of Informix 4GL code out there and Genero 4GL code which is being maintained so it would be useful to have language detection and syntax highlighting in GitHub.

## Checklist:
- [x] **I am adding a new language.**
  - [x] The extension of the new language is used in hundreds of repositories on GitHub.com.
    - Search results for each extension:
      -  https://github.com/search?utf8=%E2%9C%93&type=Code&ref=searchresults&q=extension%3A4gl+import%20fgl+NOT+nothack
      - https://github.com/search?q=extension%3Aper+layout+NOT+nothack&type=code
  - [x] I have included a real-world usage sample for all extensions added in this PR:
    - Sample source(s):
      - https://github.com/alienriver49/linguist/blob/master/samples/Genero/hello-world.4gl
      - https://github.com/alienriver49/linguist/blob/master/samples/Genero/edit-form.4gl
      - https://github.com/alienriver49/linguist/blob/master/samples/Genero%20Forms/edit-form.per
  - [x] I have included a syntax highlighting grammar: [see it in action](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_format=auto&grammar_url=https%3A%2F%2Fgithub.com%2Falienriver49%2Fgenero.tmbundle%2Fblob%2Fmaster%2Fsyntaxes%2Fgenero.tmLanguage&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2Falienriver49%2Flinguist%2Fblob%2Fmaster%2Fsamples%2FGenero%2Fedit-form.4gl&code=)

### CODE REVIEW

**Comment by lildude** [comment]
Unfortunately, popularity on GitHub for both extensions is far from meeting our requirements of:

> We try only to add languages once they have some usage on GitHub. In most cases we prefer that each new file extension be in use in hundreds of repositories before supporting them in Linguist.

... where hundreds is considered over 200 unique user/repo combos, for inclusion in the immediate future. `.4gl` comes in at only being spread across 22 unique user/repos and `.per` across 130, but quite a few of those don't look like the samples in this PR so probably not the same language and thus likely to be less popular.

**Comment by Chavi-Ansar** [comment]
Unsubcribed thanksEl 4 nov. 2020 11:43, Colin Seymour <notifications@github.com> escribió:
Unfortunately, popularity on GitHub for both extensions is far from meeting our requirements of:

We try only to add languages once they have some usage on GitHub. In most cases we prefer that each new file extension be in use in hundreds of repositories before supporting them in Linguist.

... where hundreds is considered over 200 unique user/repo combos, for inclusion in the immediate future. .4gl comes in at only being spread across 22 unique user/repos and .per across 130, but quite a few of those don't look like the samples in this PR so probably not the same language and thus likely to be less popular.

**Review by lildude — CHANGES_REQUESTED** [review]
Please can you address the conflicts and replace the hello-world.4gl sample.

Once this is good to go, I'll approve this based on the temporary requirements detailed in https://github.com/github/linguist/issues/5756

File: samples/Genero/hello-world.4gl [hunk 1]
```diff
@@ -0,0 +1,3 @@
+main
+    display "Hello, World!"
+end main
```
[the remark this answers is not in the document]
Comment (alienriver49) [reply samples/Genero/hello-world.4gl:3]: Just catching this now, I can resolve the conflicts and get a real world example added.

**Comment by alienriver49** [comment]
@lildude This has had conflicts resolved and has been updated with a better example. Let me know if there are other pieces which need to be addressed.

**Review by lildude — APPROVED** [review]

**Pull request merged**